snapshots/wb/2024-06-10/gender_statistics.feather.dvc

```diff
@@ -0,0 +1,28 @@
+meta:
+  origin:
+    # Data product / Snapshot
+    title: World Bank Gender Statistics
+    description: |-
+      The World Bank Gender Statistics dataset provides a comprehensive range of gender-related indicators grouped by various topics. These indicators are categorized under different themes such as education, employment and time use, entrepreneurship, environment, health, leadership, norms and decision-making, technology, violence, and contextual information. Each category contains numerous specific indicators, covering a wide range of issues such as literacy rates, employment by sector, legal rights, health statistics, and more. This dataset offers detailed information and insights into various aspects of gender disparity and equality across different regions and countries.
+    date_published: '2023'
```

Comment:
It says [here](https://databank.worldbank.org/source/gender-statistics#) it was updated 2024-04-15

Reply:
thank you!